### DISCUSSION TITLE
Why was 2.6.5 MySQL support not ported to 3.0?

### DISCUSSION THREAD

**Discussion by zornhsu** [Apicurio/apicurio-registry#5379] [Q&A]
Hello All, I'd like to try out mysql support in 2.6.5 on 3.0 branch, but I found out that the whole [mysql.ddl](https://github.com/Apicurio/apicurio-registry/blob/2.6.5.Final/app/src/main/resources/io/apicurio/registry/storage/impl/sql/mysql.ddl) and other files are missing. Looks like 3.0 is branched from [2.4.3](https://github.com/Apicurio/apicurio-registry/commits/0055c06029a0dc34f234f2fa967538354ca384b7/) where mysql support had not been introduced.

Why was 2.6.5 MySQL support not ported to 3.0? Where can I follow the past and future discussion on this subject?

**Comment by zornhsu** [comment]
Hello @carlesarnal , would you mind sharing your thoughts?

**Comment by EricWittmann** [comment]
Hi @zornhsu - apologies - this was purely a mistake on our end.  I didn't realize that the mysql work was added after we had already made the 3.0 split.

Would you be willing to port your work from 2.6 to 3.0?

I'm also realizing the someone else wants to have MySQL support:

* https://github.com/Apicurio/apicurio-registry/issues/5309

I definitely dropped the ball on this.  :(
